sbi/simulators/mp_simutils.py

```diff
@@ -103,16 +111,16 @@ def start_workers(simulator: Callable, num_workers: int = 1, verbose: bool = Fal
 
     Returns: queue, workers, pipes
     """
-    # try-except because start method can not be set multiple times within the same
-    # session. So, if the user runs things outside of a __main__() function, e.g. in a
-    # jupyter notebook, this would give an error from the second call of the
-    # set_start_method() function on
     try:
-        if platform == "darwin" or platform == "linux":
-            mp.set_start_method("fork")
-        elif platform == "win32":
-            mp.set_start_method("spawn")
+        start_method = dict(linux="fork", darwin="fork", win32="spawn")
+        if platform not in start_method.keys():
```

Comment:
This will happen automatically (raise KeyError) below, so you don't really need these two lines. Only if you want to be more explicit in the error message (I find that if you are in a exotic platform, it's going to be evident what the problem is). At the same time, one could bet that other exotic platforms are going to use 'fork' (just wing it, based on N=3 XD); in that case you would use in L118 below `start_method.get(platform, 'fork')`